### ISSUE TITLE
 loss appear to be 0 or nan during training

### ISSUE THREAD

**Issue by 1YuChenyao** [swc-17/SparseDrive#27]
Hello, why does the loss appear to be 0 or nan during training?
lr: 1.819e-04, eta: 2 days, 12:35:35, time: 1.072, data_time: 0.021, memory: 6349, det_loss_cls_0: 1.4239, det_loss_box_0: 2.7655, det_loss_cns_0: 0.6486, det_loss_yns_0: 0.1644, det_loss_cls_1: nan, det_loss_box_1: 0.0000, det_loss_cns_1: 0.0000, det_loss_yns_1: 0.0000, det_loss_cls_2: nan, det_loss_box_2: 0.0000, det_loss_cns_2: 0.0000, det_loss_yns_2: 0.0000, det_loss_cls_3: nan, det_loss_box_3: 0.0000, det_loss_cns_3: 0.0000, det_loss_yns_3: 0.0000, det_loss_cls_4: nan, det_loss_box_4: 0.0000, det_loss_cns_4: 0.0000, det_loss_yns_4: 0.0000, det_loss_cls_5: nan, det_loss_box_5: 0.0000, det_loss_cns_5: 0.0000, det_loss_yns_5: 0.0000, map_loss_cls_0: 0.2000, map_loss_line_0: 0.4839, map_loss_cls_1: 0.3327, map_loss_line_1: 0.4629, map_loss_cls_2: 0.3100, map_loss_line_2: 0.4467, map_loss_cls_3: 0.2659, map_loss_line_3: 0.4427, map_loss_cls_4: 0.2298, map_loss_line_4: 0.4431, map_loss_cls_5: 0.1851, map_loss_line_5: 0.4479, loss_dense_depth: 0.9466, loss: nan, grad_norm: nan

**Comment by swc-17** [comment]
How many GPUs did you use? If less than default (8 gpus), you need to lower the learning rate.

**Comment by 1YuChenyao** [comment]
> 你使用了多少个 GPU？如果少于默认值（8 个 GPU），则需要降低学习率。

Ok,I use 3 4090 GPUs.I'll try it.Thank you~

**Comment by 1YuChenyao** [comment]
> How many GPUs did you use? If less than default (8 gpus), you need to lower the learning rate.

Hello, if I use three 4090 cards, how low is the learning rate recommended to be?

**Comment by 1YuChenyao** [comment]
> How many GPUs did you use? If less than default (8 gpus), you need to lower the learning rate.

Hello, I use three 4090 and the learning rate is reduced to 1e-4, but nan and loss are still 0.
![image](https://github.com/user-attachments/assets/6e2a2a25-f375-47b0-a8d5-3aa7460d2375)

**Comment by 1YuChenyao** [comment]
> How many GPUs did you use? If less than default (8 gpus), you need to lower the learning rate.

Hello, I use three 4090 and the learning rate is reduced to 1e-4, but grad_norm: nan and loss: 0 are still appear
![image](https://github.com/user-attachments/assets/e390f208-6ce6-4ff9-8d69-b12b7a5c454b)
.
